apph-back/src/main/java/com/viseo/apph/dto/PhotoResponse.java

```diff
@@ -143,4 +145,22 @@ public PhotoResponse setLocation(Location location) {
         this.location = location;
         return this;
     }
+
+    public PhotoResponse setPhoto(Photo photo) {
+        Position position = new Position().setLat(photo.getLat())
+                .setLng(photo.getLng());
+        Location location = new Location().setAddress(photo.getAddress())
+                .setPosition(position);
+        this.setId(photo.getId())
+                .setTitle(photo.getTitle())
+                .setCreationDate(photo.getCreationDate())
+                .setModificationDate(photo.getModificationDate())
+                .setSize(photo.getSize())
+                .setTags(photo.getTags())
+                .setDescription(photo.getDescription())
+                .setShootingDate(photo.getShootingDate())
+                .setFormat(photo.getFormat())
+                .setLocation(location);
```

Comment:
Est ce qu'il a vraiment besoin de faire des set ici quand tu peux juste faire des `this.attributeName = value`

Reply:
J'ai oublié le constructeur, je vais faire autrement